version/version.go

```diff
@@ -4,7 +4,7 @@ package version
 // NOTE: remember to bump the version at the top
 // of the top-level README.md file when this is
```

Comment:
Sorry, but would it make sense now to "remember to bump the version at the top of the top-level README.md"? </snark>

Reply:
This actually got more complicated since this is on the 2.0 branch - so I need to bump the version in README.md on master, not 2.0, so it is actually seen.